### PR TITLE
Feature/shadows

### PR DESCRIPTION
**Pull request by thierryc**
Add possibility to add multiple shadows and shadow spread #277

```

<View
    name={`Swatch ${name}`}
    style={{
      height: 96,
      width: 96,
      margin: 4,
      backgroundColor: hex,
      padding: 8,
    }}
    shadows={[
      {
        shadowColor: 'lime',
        shadowOffset: { width: 0, height: 7 },
        shadowOpacity: 0.12,
        shadowRadius: 4,
      },
      {
        shadowOffset: { width: 7, height: 3 },
        shadowOpacity: 0.12,
        shadowRadius: 4,
      },
    ]}
  >

```

image

### CODE REVIEW

**Comment by thierryc** [comment]
Spread and Color 

```
const Swatch = ({ name, hex }) => (
  <View
    name={`Swatch ${name}`}
    style={{
      height: 96,
      width: 96,
      margin: 4,
      backgroundColor: hex,
      padding: 8,
      shadowColor: 'pink',
      shadowOffset: { width: 7, height: 7 },
      shadowOpacity: 0.12,
      shadowRadius: 4,
      shadowSpread: 8,
    }}
    shadows={[
      {
        shadowColor: 'lime',
        shadowOffset: { width: 0, height: 7 },
        shadowOpacity: 0.12,
        shadowRadius: 4,
      },
      {
        shadowOffset: { width: 7, height: 3 },
        shadowOpacity: 0.12,
        shadowRadius: 4,
      },
    ]}
  >
    <Text name="Swatch Name" style={{ color: textColor(hex), fontWeight: 'bold' }}>
      {name}
    </Text>
    <Text name="Swatch Hex" style={{ color: textColor(hex) }}>
      {hex}
    </Text>
  </View>
);
```

image

**Comment by thierryc** [comment]
Add possibility to add multiple shadows and shadow spread #277
Feature to support multiple shadows #161

**Comment by thierryc** [comment]
@mathieudutour any suggestions? 
Thierry

**Comment by thierryc** [comment]
Any review ?

File: src/components/ShadowsPropTypes.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import * as PropTypes from 'prop-types';
+
+export default {
+  shadowColor: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  shadowOffset: { width: PropTypes.number, height: PropTypes.number },
+  shadowOpacity: PropTypes.number,
+  shadowRadius: PropTypes.number,
+  shadowSpread: PropTypes.number,
```
Comment (mathieudutour) [inline src/components/ShadowsPropTypes.js]: I believe it's missing `shadowInner`?

Comment (thierryc) [reply src/components/ShadowsPropTypes.js]: True. I will fix that

**Pull request closed without merging**